### ISSUE TITLE
Loss calculation in prediction

### ISSUE THREAD

**Issue by gen35** [google/uis-rnn#58]
## Describe the question
When loss (or negative log likelihood) is calculated in '_update_beam_state', speaker change and assignment log likelihoods are subtracted from MSE loss. I fail to comprehend why MSE loss is treated as log likelihood of sequence generation. 

I think both transition bias and CRP alpha estimations hardly influence overall performance since log likelihoods are significantly smaller than MSE loss.
I tested decoding with entirely removed speaker change and assignment calculations and final prediction accuracy barely changed.

## My background

Have I read the `README.md` file?
* yes

Have I searched for similar questions from closed issues?
* yes

Have I tried to find the answers in the paper *Fully Supervised Speaker Diarization*?
* yes

Have I tried to find the answers in the reference *Speaker Diarization with LSTM*?
* yes

Have I tried to find the answers in the reference *Generalized End-to-End Loss for Speaker Verification*?
* yes

**Comment by AnzCol** [comment]
Yes this could happen when your likelihood dominates.  The model we design provide a generative process of speakers. And that is a crucial guarantee of the correctness of our model from a statistical perspective. Moreover, the transition bias and alpha come from your prior knowledge, and they can always change your speaker assignment when your prior is extreme: try set alpha=10^(-(10^10)) to see your prediction result.

You can somehow ‘learn’ alpha as well using empirical Bayesian methods.
